Design.md

```diff
@@ -276,3 +277,39 @@ There have been multiple rounds of CPU vulnerabilities (L1TF and MDS) which cann
 By default, Fedora CoreOS will configure the kernel to disable SMT on vulnerable machines. This conditional approach avoids incurring the performance cost on systems that aren't vulnerable. However, it fails to protect systems affected by undisclosed SMT vulnerabilities, and it allows future OS updates to disable SMT without notice if new vulnerabilities become known.
 
 We will document this policy and its consequences, and provide instructions for unconditionally enabling or disabling SMT for users who prefer a different policy.
+
+## Bucket Layout
+
+Originally discussed in [#189](https://github.com/coreos/fedora-coreos-tracker/issues/189).
+
+The `fcos-builds` bucket, fronted by http://builds.coreos.fedoraproject.org/ will be structured as follow:
+
+```
+/
+  prod/
+    streams/
+      stable/
+        stream.json
```

Comment:
As discussed OOB, let's move stream metadata to a separate directory (e.g. `/streams/stable.json`) so we can document its URL as the canonical location for stream metadata.

Reply:
OK, updated! :arrow_down: